libexec/pyenv-init

```diff
@@ -97,10 +97,7 @@ esac
 
 completion="${root}/completions/pyenv.${shell}"
 if [ -r "$completion" ]; then
-  case "$shell" in
-  fish ) echo ". '$completion'" ;;
-  *    ) echo "source '$completion'" ;;
-  esac
+  echo "source '$completion'"
```

Comment:
As of fish 2.6, both appear to work. I could have sworn with fish 2.5 that `source` was the preferred spelling.